smoke-tests/src/test/java/com/splunk/opentelemetry/TomcatSmokeTest.java

```diff
@@ -37,42 +33,12 @@
       new TomcatAttributes("9.0.40.0");
 
   private static Stream<Arguments> supportedConfigurations() {
-    return Stream.of(
-        arguments(
-            linuxImage(
-                "ghcr.io/open-telemetry/java-test-containers:tomcat-7.0.107-jdk8-20201207.405832649"),
-            TOMCAT7_SERVER_ATTRIBUTES),
-        arguments(
-            linuxImage(
-                "ghcr.io/open-telemetry/java-test-containers:tomcat-8.5.60-jdk8-20201207.405832649"),
-            TOMCAT8_SERVER_ATTRIBUTES),
-        arguments(
-            linuxImage(
-                "ghcr.io/open-telemetry/java-test-containers:tomcat-8.5.60-jdk11-20201207.405832649"),
-            TOMCAT8_SERVER_ATTRIBUTES),
-        arguments(
-            linuxImage(
-                "ghcr.io/open-telemetry/java-test-containers:tomcat-9.0.40-jdk8-20201207.405832649"),
-            TOMCAT9_SERVER_ATTRIBUTES),
-        arguments(
-            linuxImage(
-                "ghcr.io/open-telemetry/java-test-containers:tomcat-9.0.40-jdk11-20201207.405832649"),
-            TOMCAT9_SERVER_ATTRIBUTES),
-        arguments(
-            proprietaryWindowsImage("splunk-tomcat:7.0.107-jdk8-windows"),
-            TOMCAT7_SERVER_ATTRIBUTES),
-        arguments(
-            proprietaryWindowsImage("splunk-tomcat:8.5.60-jdk8-windows"),
-            TOMCAT8_SERVER_ATTRIBUTES),
-        arguments(
-            proprietaryWindowsImage("splunk-tomcat:8.5.60-jdk11-windows"),
-            TOMCAT8_SERVER_ATTRIBUTES),
-        arguments(
-            proprietaryWindowsImage("splunk-tomcat:9.0.40-jdk8-windows"),
-            TOMCAT9_SERVER_ATTRIBUTES),
-        arguments(
-            proprietaryWindowsImage("splunk-tomcat:9.0.40-jdk11-windows"),
-            TOMCAT9_SERVER_ATTRIBUTES));
+    return configurations("tomcat").otelLinux("7.0.107", TOMCAT7_SERVER_ATTRIBUTES, VMS_ALL, "8")
```

Comment:
Nit formatting:
```suggestion
    return configurations("tomcat")
        .otelLinux("7.0.107", TOMCAT7_SERVER_ATTRIBUTES, VMS_ALL, "8")
```

Reply:
This (and some other odd formatting here) was done by spotless. Before that I had them all on separate lines.

Reply:
Nothing we can do here then 🤷‍♂️